feeds/trilliumtransit.com.dmfr.json

```diff
@@ -2257,6 +2264,13 @@
         }
       ]
     },
+    {
+      "id": "f-amtrakvermonter~vt~us",
```

Comment:
let's link this to the Amtrak operator: https://www.transit.land/operators/o-9-amtrak

looks like there is already a route for Vermonter in the main feed: https://www.transit.land/routes/r-dr-vermonter

so we can mark this feed for manual imports, so it's just fetched and archived (but not imported)